tests/unit/botocore/test_endpoint_provider.py

```diff
@@ -343,3 +349,86 @@ def test_ruleset_unknown_parameter_type_raises(partitions):
             partitions=partitions,
         )
     assert "Unknown parameter type: list." in str(exc_info.value)
+
+
+@pytest.fixture()
+def empty_resolver():
+    return EndpointRulesetResolver(
+        endpoint_ruleset_data={
+            'version': '1.0',
+            'parameters': {},
+            'rules': [],
+        },
+        partition_data={},
+        service_model=None,
+        builtins={},
+        client_context=None,
+        event_emitter=None,
+        use_ssl=True,
+        requested_auth_scheme=None,
+    )
+
+
+def test_auth_schemes_conversion_sigv4(empty_resolver):
+    auth_schemes = [
+        {
+            'name': 'sigv4',
+            'signingName': 'dynamodb',
+            'signingRegion': 'my-region-1',
+            'disableDoubleEncoding': True,
+            'otherParameter': 'otherValue',
+        }
+    ]
+    at, sc = empty_resolver.auth_schemes_to_signing_ctx(auth_schemes)
+    assert at == 'v4'
+    assert sc == {
+        'region': 'my-region-1',
+        'signing_name': 'dynamodb',
+        'disableDoubleEncoding': True,
+    }
+
+
+@requires_crt()
```

Comment:
Let's update these tests to remove any CRT specific variable setting or test skipping since CRT is always available for this environment.